cross-region-replication-mm2-converter/src/main/java/com/amazonaws/services/crossregion/schemaregistry/kafkaconnect/CrossRegionReplicationMM2Converter.java

```diff
@@ -0,0 +1,157 @@
+package com.amazonaws.services.crossregion.schemaregistry.kafkaconnect;
+
+import com.amazonaws.services.schemaregistry.common.AWSSerializerInput;
+import com.amazonaws.services.schemaregistry.common.Schema;
+import com.amazonaws.services.schemaregistry.common.configs.GlueSchemaRegistryConfiguration;
+import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializationFacade;
+import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializerImpl;
+import com.amazonaws.services.schemaregistry.exception.AWSSchemaRegistryException;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializationFacade;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializerImpl;
+import com.amazonaws.services.schemaregistry.utils.AWSSchemaRegistryConstants;
+
+import kotlinx.serialization.SerializationException;
+import lombok.Getter;
+
+import org.apache.kafka.connect.data.*;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.storage.Converter;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class CrossRegionReplicationMM2Converter implements Converter {
+    @Getter
+    private GlueSchemaRegistryDeserializationFacade deserializationFacade;
+    @Getter
+    private GlueSchemaRegistrySerializationFacade serializationFacade;
+    @Getter
+    private AwsCredentialsProvider credentialsProvider;
+    @Getter
+    private GlueSchemaRegistryDeserializerImpl deserializer;
+    @Getter
+    private GlueSchemaRegistrySerializerImpl serializer;
+    @Getter
+    private boolean isKey;
+
+    /**
+     * Constructor used by Kafka Connect user.
+     */
+    public CrossRegionReplicationMM2Converter(){};
```

Comment:
Is this constructor used by Connector? if it is the case, how will the fields be initialized?

Reply:
Both constructors here are used for testing purpose